docs/user-guide/rules.md

```diff
@@ -425,18 +425,18 @@ These rules are deprecated — we won't fix bugs nor add options, and we will re
+| Rule                                                                                                                                                                                                                                           | ⭐️ | 💅  | 🔧  |
+| ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :-: | :-: | :-: |
+| [`selector-attribute-brackets-space-inside`](../../lib/rules/selector-attribute-brackets-space-inside/README.md)<br/>Require a single space or disallow whitespace on the inside of the brackets within attribute selectors.                   |     |     | ✅  |
+| [`selector-attribute-operator-space-after`](../../lib/rules/selector-attribute-operator-space-after/README.md)<br/>Require a single space or disallow whitespace after operators within attribute selectors.                                   |     |     | ✅  |
+| [`selector-attribute-operator-space-before`](../../lib/rules/selector-attribute-operator-space-before/README.md)<br/>Require a single space or disallow whitespace before operators within attribute selectors.                                |     |     | ✅  |
+| [`selector-combinator-space-after`](../../lib/rules/selector-combinator-space-after/README.md)<br/>Require a single space or disallow whitespace after the combinators of selectors.                                                           |     |     | ✅  |
+| [`selector-combinator-space-before`](../../lib/rules/selector-combinator-space-before/README.md)<br/>Require a single space or disallow whitespace before the combinators of selectors.                                                        |     |     | ✅  |
+| [`selector-descendant-combinator-no-non-space`](../../lib/rules/selector-descendant-combinator-no-non-space/README.md)<br/>Disallow non-space characters for descendant combinators of selectors.                                              |     |     | ✅  |
+| [`selector-max-empty-lines`](../../lib/rules/selector-max-empty-lines/README.md)<br/>Limit the number of adjacent empty lines within selectors.                                                                                                |     |     | ✅  |
+| [`selector-pseudo-class-case`](../../lib/rules/selector-pseudo-class-case/README.md)<br/>Specify lowercase or uppercase for pseudo-class selectors.                                                                                            |     |     | ✅  |
+| [`selector-pseudo-class-parentheses-space-inside`](../../lib/rules/selector-pseudo-class-parentheses-space-inside/README.md)<br/>Require a single space or disallow whitespace on the inside of the parentheses within pseudo-class selectors. |     |     | ✅  |
```

Comment:
Good catch!

I had actually expected this to be caught by the existing tests.
Nice that there is extra coverage now.

Reply:
Ya, this new case is what we had to add first. 😅